lib/matplotlib/colorbar.py

```diff
@@ -238,6 +238,8 @@ def __init__(self, colorbar):
         super().__init__(nbins=nbins, steps=steps)
 
     def tick_values(self, vmin, vmax):
```

Comment:
why is the whole function not just
```
return super().tick_values(
    *np.clip([vmin, vmax], self._colorbar.norm.vmin, self._colorbar.norm.vmax))
```
(possibly spread over more lines)?
In particular why does this need to implement its own culling of out of bounds ticks?

Reply:
Because the extended part of the colorbar makes the colorbar vmin/ vmax bigger than what we want ticked.  Ie we don’t want ticks in the extended portion of the axis.

Reply:
ugh, sure.